src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -20,104 +24,81 @@ class RegistrationServiceImplTest {
     static void beforeAll() {
         storageDao = new StorageDaoImpl();
         registrationService = new RegistrationServiceImpl();
-
     }
 
     @BeforeEach
     void beforeEach() {
         user = new User();
+        user.setLogin(DEFAULT_LOGIN);
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
     }
 
     @Test
     void register_userLoginOnNull_notOk() {
-        user.setAge(23);
-        user.setPassword("qwer1234");
+        user.setLogin(null);
         Exception exception = assertThrows(InvalidUserDataException.class,
```

Comment:
Exception exception = assertThrows(InvalidUserDataException.class,
                () -> registrationService.register(user), "Login can't be empty!");

==

assertThrows(InvalidUserDataException.class,
                () -> registrationService.register(user), "Login can't be empty!");
                
                same to all methods below